### PR TITLE
feat: rosetta-cli check:construction MA workflow

### PR DESCRIPTION
**Pull request by AlanVerbner**
# Description

This PR implements #274 

# Proposed Solution

Checkout the `README`

# Important Changes Introduced

N/A

# Testing

Steps are explained in the `README`

### CODE REVIEW

File: test/check/workflows/transfer-ma.ros [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+    // The suggested_fee is returned in the /construction/metadata
+    // response and saved to transfer_dry_run.suggested_fee.
+    suggested_fee = find_currency_amount({
+      "currency":{{currency}},
+      "amounts":{{transfer_dry_run.suggested_fee}}
+    });
+
+    // We can access the variables of other scenarios, so we don't
+    // need to recalculate raw_change_amount.
+    change_amount = {{raw_change_amount}} - {{suggested_fee.value}};
```
Comment (james-iohk) [inline test/check/workflows/transfer-ma.ros:213]: Consider making the change amount so the only remainder is the suggested_fee. Currently there is an excess of 6ADA due to raw_change_amount being high. Seeing as the outputs are hardcoded in this workflow we could do something like `change_amount = {{sender.balance.value}} - {{token_bundle_min_ADA}} * 2 - {{suggested_fee.value}};`

File: test/check/workflows/transfer-ma.ros [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+  },
+
+  // Create a separate scenario to request funds so that
+  // the address we are using to request funds does not
+  // get rolled back if funds do not yet exist.
+  request{
+    loaded_account = find_balance({
+      "account_identifier": {{random_account.account_identifier}},
+      "minimum_balance":{
+        "value": "10000000",
```
Comment (james-iohk) [inline test/check/workflows/transfer-ma.ros:22]: This value is too low. We need at least 16.5ADA at the moment due to high raw change. Change this to 20ADA for a quick fix.

**Comment by disassembler** [comment]
bors try

**Comment by disassembler** [comment]
bors ping

**Comment by disassembler** [comment]
bors try

**Review by james-iohk — APPROVED** [review]

**Pull request merged**